test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -4005,6 +4017,48 @@ TEST_F(AtenXlaTensorTest, TestEinsumPyTorchLowerRepeatedAxisBackward) {
   });
 
   ExpectCounterNotChanged("aten::.*", cpp_test::GetIgnoredCounters());
+  ExpectCounterChanged("EinsumFallback", cpp_test::GetIgnoredCounters());
+  ExpectCounterChanged("xla::einsum", cpp_test::GetIgnoredCounters());
+}
+
+TEST_F(AtenXlaTensorTest, TestEinsumThreeInputs) {
+  torch::Tensor x =
```

Comment:
can we add another test to reflect other einsum that will fail `EquationIsValid` check?  This one fails `tensors.size() > 2` check I believe

Reply:
TestEinsumPyTorchLowerRepeatedAxis exercises the code where EquationIsValid returns false.